cms/standard_pages/models.py

```diff
@@ -38,3 +47,76 @@ class InformationPage(BasePage):  # type: ignore[django-manager-missing]
         index.SearchField("summary"),
         index.SearchField("content"),
     ]
+
+
+class MethodologyPage(BasePage):  # type: ignore[django-manager-missing]
+    parent_page_types: ClassVar[list[str]] = ["topics.TopicPage"]
+
+    template = "templates/pages/methodology_page.html"
+
+    summary = RichTextField(features=settings.RICH_TEXT_BASIC)
+    published_date = models.DateField()
+    last_revised_date = models.DateField(blank=True, null=True)
```

Comment:
nitpick: `publication_date` would be better as `publication_date` (for consistency with other models)

Reply:
Done [7bad8a3](https://github.com/ONSdigital/dis-wagtail/pull/57/commits/7bad8a3ecef388d5ea915f5db34c56fefc4f949a)